client.js

```diff
@@ -24,25 +28,41 @@ let client;
 const bigBucketOfData = {};
 const ring = new Set([]);
 let lastNode;
+const NULL_USER = { id: null };
 
-function fetch({ _: args }) {
-  if (!args[0]) {
+async function lookup({ _, ...rest }) {
+  if (!rest.id) {
     console.log("fetch required an ID");
     process.exit();
   }
-  id = parseInt(process.argv[3], 10);
-  client.fetch({ id }, (err, user) => {
-    if (err) {
-      console.log(err);
-    } else {
-      console.log(user);
-    }
-  });
+  console.log("Beginning client-side lookup: ", rest.id);
+  const lookedUpUser = await client.lookup({ id: rest.id });
+  console.log(lookedUpUser);
+  if (!lookedUpUser) {
```

Comment:
When I fetch someone that doesn't exist, I get an empty object back. This evaluates to truthy in JS, so I never see the found nothing message. I suggest testing for lookedUpUser.id or something. A better solution would be to return the gRPC code for does not exist, but we can defer until later.